apps/src/gamelab/GameLab.js

```diff
@@ -174,7 +256,7 @@ GameLab.prototype.callHandler = function (name, allowQueueExtension, extraArgs)
     } else {
       // TODO (cpirich): support events with parameters
       if (handler.name === name) {
-        this.JSInterpreter.queueEvent(handler.func, extraArgs);
+        handler.func.apply(null, extraArgs);
```

Comment:
When we un-revert, it would also be great if we added a test that would have caught this :)